modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectDefinitionVulcanBatchEngineTaskItemDelegateTest.java

```diff
@@ -182,7 +182,13 @@ private ObjectDefinition _createObjectDefinition(String name) {
 					RandomTestUtil.randomString());
 				id = RandomTestUtil.randomLong();
 				label = Collections.singletonMap("en_US", "O" + sanitizedName);
-				modifiable = true;
+
+				if (!GetterUtil.getBoolean(
```

Comment:
I think this condition is wrong, it must validate if the feature flag is active/true